packages/main/src/themes/Input.css

```diff
@@ -1,5 +1,6 @@
 @import "./FormComponents.css";
 @import "./InvisibleTextStyles.css";
+@import "./InputSharedStyles.css";
```

Comment:
This is a big change and my opinion on that is that it should not be done here as part of the icon styling as it does not have have a direct relationship. No need to complicate the change and have the risk of potentially introducing issues in all inputs for the release as it would be impossible to test all input components thoroughly in a short matter.

It is the correct way to go, but not as a 2.0 change.

Reply:
I though it would be good since we are doing "css refactoring" somehow by moving all icon related styles to separate file. Basically the file contain only 1:1 css code that is duplicated in both files (Input.css and InputSharedStyles.css). If you do global search in your IDE for the file content of InputSharedStyles it will match the content in Input.css.